api/lib/gateways/Academy-Benefits/Search.js

```diff
@@ -47,24 +49,27 @@ module.exports = options => {
         equal = false;
         break;
       }
-    };
+    }
     return equal;
-  }
+  };
 
   const recordEquality = (record1, record2) => {
-    return record1.id === record2.id
-      && stringDataEquality(record1.firstName, record2.firstName)
-      && stringDataEquality(record1.lastName, record2.lastName)
-      && stringDataEquality(record1.nino, record2.nino)
-      && stringDataEquality(record1.postcode, record2.postcode)
-      && (record1.dob === record2.dob || moment(record1.dob).diff(moment(record2.dob), 'days') < 1)
-  }
+    return (
```

Comment:
Why has the return value been put in brackets?

Reply:
I didn't make changes to this file, but I'm guessing it's to help the code to look more tidy instead of being in one loong line, could be wrong though